src/iperf_udp.c

```diff
@@ -204,6 +207,32 @@ iperf_udp_recv(struct iperf_stream *sp)
     return r;
 }
 
+/* Determine send size for a single packet */
+static int iperf_udp_send_size(struct iperf_stream *sp)
+{
+    int class_sz;
+    int r;
+
+    if (!sp->settings->imix)
+        return sp->settings->blksize;
+
+    /* IMIX (according to wikipedia) says:
+     *   Packet size (incl. IP header) 	# Packets 	Distribution (in packets)
+     *                40 	            7 	              58.333333%
+     *                576 	            4 	              33.333333%
+     *                1500 	            1 	              8.333333%
+     */
+    r = random() % 1000;
```

Comment:
Is the Random seed set explicitly?
I am wondering what is the behaviour do we want ?

if we set a static seed (or give control of setting a seed), then successive runs are identical and conform to the packet size mixture distribution.

Without that every run is different, giving seed explicit control to the user will enable both behaviors.

Reply:
Seed is set lower in file (`srandom(0xDEADF00D);`). Would prefer to avoid another config flag (already quite a few) but don't mind adding if necessary